mothertongues/config/models.py

```diff
@@ -32,7 +32,7 @@ class BaseConfig(BaseModel):
     model_config = ConfigDict(extra="forbid")
 
 
-class Audio(BaseConfig):
+class Audio(BaseConfig, extra="allow"):
```

Comment:
Huh, I haven't seen this syntax, is this any different than just adding:

`model_config = ConfigDict(extra='allow')` ?

Reply:
Oh!  In fact this syntax is somewhat obscure, and I'm struggling to find a reference for it aside from https://stackoverflow.com/questions/69617489/can-i-get-incoming-extra-fields-from-pydantic

Reply:
Since it doesn't appear to be (very) documented I've switched back to the normal way of doing it!